cmake/onnxruntime.cmake

```diff
@@ -66,6 +66,7 @@ target_link_libraries(onnxruntime PRIVATE
     ${PROVIDERS_NUPHAR}
     ${PROVIDERS_DML}
     ${PROVIDERS_ACL}
+    ${onnxruntime_winml}
```

Comment:
>onnxruntime_winml [](start = 6, length = 17)

Should the naming have adapter / core in there? I had to check in winml.cmake to double check which lib(s) were getting included